exercises/03-Polling-SCM/README.md

```diff
@@ -1,8 +1,96 @@
-**Creating a Freestyle Project**
+**Polling SCM for Changes**
 =====================================================
 
+Now that you have your build steps, it's time to learn how to automate running them!
 
+Like most CI build systems, Jenkins is intended to be used to trigger builds when new changes are found in Source Code Management (SCM).
 
-**Step 1: Setup a Freestyle Project**
+In this exercise, you'll setup a Git repository and poll it for changes. If any are made, it will trigger the jobs you made in the last exercise.
+
+**Step 1: Setup a Git Repository**
+-------------------------------------------
+
+First, you'll need to have a Git repository to make changes to. You're free to make one of your own, or clone the [Demo Repo from my account](https://github.com/devbyaccident/Demo_Repo).
```

Comment:
What caught my eye was using the word `clone` - are you using that for an audience that would be unfamiliar with the term `fork`?

The other thing I'm thinking about - and take this with the grain of salt that I'm commenting before reading the rest of this page - shouldn't this really say `git repo on github` instead of just `git repo`? just trying to draw the distinction/ make sure it's clear that you're not talking about a locally cloned repo (yet) and instead you're talking about something on github

Reply:
> What caught my eye was using the word clone - are you using that for an audience that would be unfamiliar with the term fork?

Good catch, thanks,

> The other thing I'm thinking about - and take this with the grain of salt that I'm commenting before reading the rest of this page - shouldn't this really say git repo on github instead of just git repo?

That might be adding too much additional info. Trying to keep this simple to start.